build/nuke/Build.Steps.cs

```diff
@@ -172,20 +172,20 @@ partial class Build
                     .SetFramework(framework)
                     .SetOutput(TracerHomeDirectory / framework)));
 
-            // StartupHook is supported starting .Net Core 3.1.
-            // We need to emit StartupHook and ClrProfilerManagedLoader assemblies only for .NET Core 3.1 target framework.
+            // AutoInstrumentationStartupHook is supported starting .Net Core 3.1.
```

Comment:
This refers to the StartupHook as a .NET concept and shouldn't be changed.

Reply:
fixed 3cf27c63d4204c6c279def22755c849db1ed03e0